views/templates/PrintTest/render.tpl

```diff
@@ -1,10 +1,16 @@
 <?php
-use oat\tao\helpers\Layout;
 use oat\tao\helpers\Template;
+use oat\tao\model\layout\AmdLoader;
+
+$configUrl    = get_data('client_config_url');
+$requireJsUrl = Template::js('lib/require.js', 'tao');
+$bootstrapUrl = Template::js('loader/bootstrap.js', 'tao');
+
+$loader = new AmdLoader($configUrl, $requireJsUrl, $bootstrapUrl);
 ?><!doctype html>
 <html>
 <head>
-    <?= Layout::getAmdLoader(Template::js('loader/taoBooklet.min.js', 'taoBooklet'), 'taoBooklet/controller/PrintTest/render', get_data('client_params'), true) ?>
+    <?= $loader->getBundleLoader(Template::js('loader/taoBookletRunner.min.js', 'taoBooklet'), 'taoBooklet/controller/PrintTest/render', get_data('client_params')); ?>
```

Comment:
Would not be better maybe inject the result of those call in the template instead of instantiate all here?

```suggestion
    <?= $headers; ?>
```